#! /bin/sh /usr/share/dpatch/dpatch-run
## /tmp/programs--eroute--eroute.8-linebreak.patch.dpatch by  <jari.aalto@cante.net>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP:
## DP: Author: Jari Aalto <jari.aalto@cante.net>
## DP:
## DP:
## DP: groff warning [p 1, 1.5i]: can't break line

@DPATCH@

diff --git a/programs/eroute/eroute.8 b/programs/eroute/eroute.8
index 2c36b0b..4b1ee90 100644
--- a/programs/eroute/eroute.8
+++ b/programs/eroute/eroute.8
@@ -22,11 +22,11 @@
 ipsec_eroute \- manipulate IPSEC extended routing tables
 .SH "SYNOPSIS"
 .HP \w'\fBipsec\ eroute\fR\ 'u
-\fBipsec eroute\fR \fB\fI\-\-add\fR\fR\ \fB\-\-eraf\fR\ \fI(inet\ |\ inet6)\fR\ \fB\-\-src\fR\ \fIsrc/srcmaskbits|srcmask\fR\ \fB\-\-dst\fR\ \fIdst/dstmaskbits|dstmask\fR\ [[\fB\-\-transport\-proto\fR\ \fItransport\-protocol\fR]]\ [\fB\-\-src\-port\ \fR\fB\fIsource\-port\fR\fR]\ [\fB\-\-dst\-port\ \fR\fB\fIdest\-port\fR\fR]\ [<SAID>] 
+\fBipsec eroute\fR \fB\fI\-\-add\fR\fR \fB\-\-eraf\fR \fI(inet | inet6)\fR \fB\-\-src\fR \fIsrc/srcmaskbits|srcmask\fR \fB\-\-dst\fR \fIdst/dstmaskbits|dstmask\fR [[\fB\-\-transport\-proto\fR \fItransport\-protocol\fR]] [\fB\-\-src\-port \fR\fB\fIsource\-port\fR\fR] [\fB\-\-dst\-port \fR\fB\fIdest\-port\fR\fR] [<SAID>]
 .HP \w'\fBipsec\ eroute\fR\ 'u
-\fBipsec eroute\fR \fB\fI\-\-replace\fR\fR\ \fB\-\-eraf\fR\ \fI(inet\ |\ inet6)\fR\ \fB\-\-src\fR\ \fIsrc/srcmaskbits|srcmask\fR\ \fB\-\-dst\fR\ \fIdst/dstmaskbits|dstmask\fR\ [[\fB\-\-transport\-proto\fR\ \fItransport\-protocol\fR]]\ [\fB\-\-src\-port\ \fR\fB\fIsource\-port\fR\fR]\ [\fB\-\-dst\-port\ \fR\fB\fIdest\-port\fR\fR]\ [<SAID>] 
+\fBipsec eroute\fR \fB\fI\-\-replace\fR\fR \fB\-\-eraf\fR \fI(inet | inet6)\fR \fB\-\-src\fR \fIsrc/srcmaskbits|srcmask\fR \fB\-\-dst\fR \fIdst/dstmaskbits|dstmask\fR [[\fB\-\-transport\-proto\fR \fItransport\-protocol\fR]] [\fB\-\-src\-port \fR\fB\fIsource\-port\fR\fR] [\fB\-\-dst\-port \fR\fB\fIdest\-port\fR\fR] [<SAID>]
 .HP \w'\fBipsec\ eroute\fR\ 'u
-\fBipsec eroute\fR \fB\fI\-\-del\fR\fR\-\-del\fB\-\-eraf\fR\ \fI(inet\ |\ inet6)\fR\ \fB\-\-src\fR\fIsrc/srcmaskbits|srcmask\fR\fB\-\-dst\fR\fIdst/dstmaskbits|dstmask\fR\ [[\fB\-\-transport\-proto\fR\ \fItransport\-protocol\fR]]\ [\fB\-\-src\-port\ \fR\fB\fIsource\-port\fR\fR]\ [\fB\-\-dst\-port\ \fR\fB\fIdest\-port\fR\fR]\ [<SAID>] 
+\fBipsec eroute\fR \fB\fI\-\-del\fR\fR\-\-del\fB\-\-eraf\fR \fI(inet | inet6)\fR \fB\-\-src\fR\fIsrc/srcmaskbits|srcmask\fR\fB\-\-dst\fR\fIdst/dstmaskbits|dstmask\fR [[\fB\-\-transport\-proto\fR \fItransport\-protocol\fR]] [\fB\-\-src\-port \fR\fB\fIsource\-port\fR\fR] [\fB\-\-dst\-port \fR\fB\fIdest\-port\fR\fR] [<SAID>]
 .HP \w'\fBipsec\ eroute\fR\ 'u
 \fBipsec eroute\fR \-\-clear
 .HP \w'\fBipsec\ eroute\fR\ 'u
